### PR TITLE
Update of error codes in Articles controller

### PR DESCRIPTION
**Pull request by shsteindorsson**
Update of error codes in Articles controller
================================
[PT-link](https://www.pivotaltracker.com/story/show/169548013)
## PR content
In this PR we are updating status codes for error messages in Articles controller, for `index` and `show` actions.  
Message for successfully editing an article is also updated.

### CODE REVIEW

File: app/controllers/v1/articles_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def index
     articles = Article.all
     
     if articles.empty? 
-      render_error_message("There are no articles here", 200)
+      render_error_message("There are no articles here", 401)
```
Comment (GergKllai1) [inline app/controllers/v1/articles_controller.rb:8]: Why did you take the approach to send back an error when the database is empty?

Comment (lucamarial) [reply app/controllers/v1/articles_controller.rb:8]: We have written our code to catch an error, thats why we chose an error status

Comment (shsteindorsson) [reply app/controllers/v1/articles_controller.rb:8]: My teammates had some heated discussions about this yesterday... this was what the jury decided

Comment (faraznaeem) [reply app/controllers/v1/articles_controller.rb:8]: Did you consider that this is not an error but a successful response with no articles?

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**